lib/rucio/core/transfer.py

```diff
@@ -889,22 +896,26 @@ def __set_transfer_state(external_host, transfer_id, new_state, session=None):
 
 
 @read_session
-def __get_unavailable_read_rse_ids(session=None):
+def __get_unavailable_rse_ids(activity, session=None):
```

Comment:
This should be called `operation` not `activity`, since it might be confusing with the transfer activities. In other places in the code we already call this operation.

Reply:
Fixed